apps/openmw/mwmechanics/character.cpp

```diff
@@ -255,7 +255,7 @@ void CharacterController::refreshHitRecoilAnims()
                 && mAnimation->hasAnimation("knockout"))
         {
             mTimeToWake = time(NULL);
-            mTimeToWake += rand() % 2 + 1; // Wake up after 1 to 3 seconds
+            mTimeToWake += (int)( (float)rand() / (float)RAND_MAX * 2) + 1; // Wake up after 1 to 3 seconds
```

Comment:
This still has not fixed the problem. This expression only gives 3 when rand() == RAND_MAX, which is an extremely unlikely occurrence. You need to obtain a random number in the range [0,2] and add 1 to it to get the desired result. Also see [here](http://en.cppreference.com/w/cpp/numeric/random/uniform_int_distribution) for random integers in C++11.